SimDataFormats/TrackerDigiSimLink/interface/PixelSimHitAddExtraInfo.h

```diff
@@ -0,0 +1,41 @@
+#ifndef PixelSimHitAddExtraInfo_h
+#define PixelSimHitAddExtraInfo_h
+
+#include "DataFormats/GeometryVector/interface/LocalPoint.h"
+#include <vector>
+#include <cstdint>
+
+class PixelSimHitAddExtraInfo {
+public:
+  PixelSimHitAddExtraInfo(size_t Hindex, Local3DPoint entryP, Local3DPoint exitP, unsigned int ch) {
+    index_ = Hindex;
+    theEntryPoint_ = entryP;
+    theExitPoint_ = exitP;
+    chan_.push_back(ch);
+  };
+  PixelSimHitAddExtraInfo(){};
+  ~PixelSimHitAddExtraInfo(){};
```

Comment:
could be `default`

Reply:
what do you mean by "could be default" ? that the line 17 could be removed ?

Reply:
ah, I thought lines 16 and 17 were the default c'tor and d'tor... gloups...

Reply:
@mmusich I am sorry, but I have really no idea how I should change the code to create the default constructor and destructor in place of lines 16 and 17.  Could you give me an example ?

Reply:
sure: https://en.cppreference.com/w/cpp/language/default_constructor